karma.conf.single-run.js

```diff
@@ -55,7 +55,7 @@ module.exports = function karmaConfig(config) {
                 postLoaders: [
                     {
                         test: /\.jsx?$/,
-                        exclude: /(__tests__|node_modules|legacy|libs\\Cesium)\\|(__tests__|node_modules|legacy|libs\/Cesium)\/|webpack\.js|utils\/(openlayers|leaflet)/,
+                        exclude: /(__tests__|node_modules|legacy|libs\\Cesium|libs\\html2canvas)\\|(__tests__|node_modules|legacy|libs\/Cesium|libs\/html2canvas)\/|webpack\.js|utils\/(openlayers|leaflet)/,
```

Comment:
Can you download the release as cesium does, using mvn.
See this https://github.com/geosolutions-it/MapStore2/blob/master/web/pom.xml#L69